docs/pages/build-reference/app-versions.md

```diff
@@ -0,0 +1,110 @@
+    }
+  }
+}
+```
+
+</Collapsible>
+
+<Collapsible summary="Local version source">
+
+With this **eas.json**, the version for all builds will be based on the value from **app.json** or native code. When you build using `production` profile, the version will be incremented in the local code before the build.
+
+```json
+{
+  "cli": {
+    "appVersionSource": "local"
+  },
+  "build": {
+    "staging": {
+      "distribution": "internal",
+      "android": {
+        "buildType": "apk"
+      }
+    },
+    "production": {
+      "autoIncrement": true
+    }
+  }
+}
+```
+
+</Collapsible>
+
+## Remote version source
+
+You can configure your project to rely on EAS servers to store and manage the version of your app, to do that add `{ "cli": { "appVersionSource": "remote" } }` in your **eas.json**. The remote version will be initialized with the value from a local project — if you would like to explicitly set the value directly, or EAS CLI is not able to detect what version an app is on, you can use the `eas build:version:set` command. EAS stores version information scoped by account, slug, platform, and application ID/bundle identifier — so, for example, if you are building variants with different application IDs or bundle identifiers, versioning will be independent for each of them.
+
+If you want to build your project locally in Android Studio or Xcode using the same version stored remotely on EAS, you can update your local project with the remote versions using `eas build:version:sync`.
+
+Enabling the `autoIncrement` option in the remote app version source mode is currently only available for `versionCode`/`buildNumber`. 
+
+When using a remote app version source, the values in **app.json** will not be updated when the version is incremented remotely, and so the local and remote values will fall out of sync. The remote source values will be set on the native project when running a build, and they are the source of truth  — however, the values specified in your **app.json** will be present in `Constants.expoConfig` and `Constants.manifest` exposed by `expo-constants`. Use `expo-application` to determine your application version at runtime instead, and remove `versionCode`/`buildNumber` from your **app.json**.
+
+### Limitations
+
+- `eas build:version:sync` command on Android does not support bare projects with multiple flavors, but the rest of the remote versioning functionality should work with all projects.
+- `autoIncrement` does not support the `version` option.
+- It's not supported if you are using EAS Update and runtime policy set to `"runtimeVersion": { "policy": "nativeVersion" }`. For similar behavior, use the `"appVersion"` policy instead.
+
+### Recommended workflow
+
+The main goal of this feature is to avoid manual changes to the project every time you are uploading a new archive to run it on TestFlight / Play Store testing channels. When you are doing a production release, the user-facing version change should be explicit.
```

Comment:
```suggestion
The main goal of this feature is to avoid manual changes to the project every time you are uploading a new archive to run it on TestFlight/Play Store testing channels. When you are doing a production release, the user-facing version change should be explicit.
```

Let's refrain ourselves from adding extra spacing before and after a "/".  😄